Build : error TS7056: The inferred type of this node exceeds the maximum length

Hello

Hi, I get this error when I try to build the library, do you get it too?

```sh
pnpm run build

> sortablejs-vue3@1.2.7 build sortablejs-vue3
> vite build && vue-tsc --emitDeclarationOnly --project tsconfig.dist.json && mv dist/lib dist/types && rm -rf dist/favicon.ico

vite v4.0.4 building for production...
✓ 4 modules transformed.
dist/sortablejs-vue3.es.js  3.53 kB │ gzip: 1.16 kB
dist/sortablejs-vue3.umd.js  2.00 kB │ gzip: 0.99 kB
src/components/Sortable.vue:6:1 - error TS7056: The inferred type of this node exceeds the maximum length the compiler will serialize. An explicit type annotation is needed.

  6 type SortableOptionsProp = Omit<
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  7   SortableOptions | AutoScrollOptions,
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
...
134 });
    ~~~
135 </script>

Found 1 error in src/components/Sortable.vue:6
```

Regards

Mathieu

--- a/src/components/Sortable.vue
+++ b/src/components/Sortable.vue
@@ -19,6 +19,12 @@ type SortableOptionsProp = Omit<
   | "onChange"
 >;
 
+type ExposedProps = {
+  containerRef: Ref<HTMLDivElement | null>;
+  sortable: Ref<Sortable | null>;
+  isDragging: Ref<boolean>;
+}
+
 const props = defineProps({
   /** All SortableJS options are supported; events are handled by the `defineEmits` below and should be used with v-on */
   options: {
@@ -76,9 +82,9 @@ const getKey = computed(() => {
 
 defineExpose({
   containerRef,
-  sortable: <Ref<Sortable | null>>sortable,
+  sortable,
   isDragging,
-});
+} as ExposedProps);
 
 watch(containerRef, (newDraggable) => {
   if (newDraggable) {

